app/src/widgets/dishcard.tsx

```diff
@@ -1,18 +1,65 @@
-import '../styles/index.css'
+import "../styles/index.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import mug from "../assets/mug_icon_contained.svg";
+import container from "../assets/dish_icon_contained.svg";
+import { config } from "../config";
 
-export default ({dish}) => {
-    const dateFormat = { day: 'numeric', year: 'numeric', month: 'short'}
-    return (
-        <div className="dish-card mb-3">
-            <div className="type-icon">
-                <img />
-            </div>
-            <div className="d-flex flex-column">
-                <p className="small-text-1 m-0">{dish.dishID} # {dish.id}</p>
-                {/* <p className="details-1 m-0">Return before {dish.due.toLocaleDateString("en-US", dateFormat)}</p>
-                <p className="small-text-1 m-0">{dish.type} # {dish.id}</p>
-                <p className="details-2 m-0">Checked out {dish.checkedOut.toLocaleDateString("en-US", dateFormat)}</p> */}
-            </div>
-        </div>
-    )
-}
\ No newline at end of file
+export default ({ dish, token }) => {
+  const options = { weekday: "long", year: "numeric", month: "long" };
+  const [dishAPI, setDishAPI] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(`${config.serverUrl}/api/dish`, {
+        headers: { "x-api-key": config.apiKey, "session-token": token },
+        params: { id: dish.dish },
+      })
+      .then(function (response) {
+        setDishAPI(response.data.dish);
+      })
+      .catch(function (error) {
+        console.log(error);
+      });
+  }, []);
+
+  // console.log(dishAPI['type'])
+
+  if (dishAPI["type"] === "mug") {
+    var icon = mug;
```

Comment:
don't use `var` anymore, it's super outdated. `const` here or `let` if you need to reassign the variable